paper-server/patches/sources/net/minecraft/server/network/ServerGamePacketListenerImpl.java.patch

```diff
@@ -2471,6 +2471,19 @@
          this.player.resetLastActionTime();
          ServerLevel serverLevel = this.player.serverLevel();
          BlockPos pos = packet.getPos();
+         if (serverLevel.hasChunkAt(pos)) {
++            // Paper start - Add API for client-side signs
++            if (!new io.papermc.paper.event.packet.UncheckedSignChangeEvent(this.player.getBukkitEntity(),
```

Comment:
why is this event always called, shouldn't it be only called if the if below is false, as in, the server doesnt know a sign there? the method and the event are connected and should only work together the way I understand it, plus the the naming can reflect that